.eslintrc.js

```diff
@@ -165,25 +165,7 @@ module.exports = {
       },
     ],
     "unicorn/prevent-abbreviations": "off",
-    "import/order": [
```

Comment:
This was preventing me from committing since it had an opposite opinion from Prettier